HousingSearchApi/V1/Infrastructure/Factories/AssetQueryGenerator.cs

```diff
@@ -22,6 +22,15 @@ public class AssetQueryGenerator : IQueryGenerator<QueryableAsset>
 
         public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescriptor<QueryableAsset> q)
         {
+            //Pre-default check for relationships query
+            if (request.GetType() == typeof(GetAssetRelationshipsRequest))
+            {
+                var relationshipRequest = request as GetAssetRelationshipsRequest;
+                return _queryFilterBuilder
+                    .WithExactQuery(relationshipRequest.SearchText, new List<string> { "parentAssetIds" })
```

Comment:
Have you tested this? Doesn't the field contain '#' to separate the ids, meaning an exact match wouldn't work?

Reply:
I have tested it - and I've included E2E tests to that effect - you misunderstand what an exact query is in ElasticSearch. Say my query is "12 Pitcairn" and the field list is "address1"
Exact query means find "12 Pitcairn" in the field address1 / if a document contains the phrase "12 Pitcairn" anywhere in the field, return the document
Wildstar query means find either 12, or Pitcairn in address1 / if a document contains any of these, return the document